roles/kubernetes/master/tasks/kubeadm-setup.yml

```diff
@@ -26,19 +9,8 @@
   file:
     path: "{{ kube_config_dir }}/admin.conf"
     state: absent
-  when: not kubeadm_already_run.stat.exists
-
-- name: kubeadm | Delete old static pods
```

Comment:
We don't want to break upgrades from non-kubeadm, I think we should leave this block in